lib/aws/active_job/sqs/configuration.rb

```diff
@@ -157,15 +243,15 @@ def config_file
         # Load options from YAML file
         def load_from_file(file_path)
           opts = load_yaml(file_path) || {}
+          opts[:queues]&.each_key do |queue|
+            opts[:queues][queue] = { url: opts[:queues][queue] } if opts[:queues][queue].is_a?(String)
```

Comment:
Is this for backwards compatibility? I think it's fair to make this the ideal case and not worry about this